<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RC4</class>
 <widget class="QMainWindow" name="RC4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RC4</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="toolTip">
         <string>Choose Encryption Algorithm</string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <item>
         <property name="text">
          <string>RC4</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_encrypt">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Encrypt Input Text</string>
        </property>
        <property name="text">
         <string>Encrypt</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_decrypt">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Decrypt input text</string>
        </property>
        <property name="text">
         <string>Decrypt</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>Encryption/Decyption in Hexadecimal (default in binary)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="Cipher">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Key : </string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="lineEdit_CC_key"/>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Range ( 0 : 26 )</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>481</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Enter the key below in binary or Hexadecimal </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_rc4_key">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>561</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>  Plain Text in Binary or Hexadecimal </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit_plain"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>  Encrypted Text in Binary or Hexadecimal </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit_encrypted"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Steps of Encryption/Decyption:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>24</height>
    </rect>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
    <addaction name="actionGo_Back_to_Main_Menu"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionReset_Fields"/>
    <addaction name="separator"/>
    <addaction name="actionClear_Key"/>
    <addaction name="actionClear_Plain_Text"/>
    <addaction name="actionClear_Encrypted_Text"/>
    <addaction name="actionClear_Steps"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionReset_Fields">
   <property name="text">
    <string>Reset Fields</string>
   </property>
  </action>
  <action name="actionClear_Plain_Text">
   <property name="text">
    <string>Clear Plain Text</string>
   </property>
  </action>
  <action name="actionClear_Encrypted_Text">
   <property name="text">
    <string>Clear Encrypted Text</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionGo_Back_to_Main_Menu">
   <property name="text">
    <string>Go Back to Main Menu</string>
   </property>
  </action>
  <action name="actionClear_Key">
   <property name="text">
    <string>Clear Key</string>
   </property>
  </action>
  <action name="actionClear_Steps">
   <property name="text">
    <string>Clear Steps</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
